src/components/PastFiles.js

```diff
@@ -1,5 +1,122 @@
+  const [pastFiles, setPastFiles] = useState(files);
+
+  function toggleView(index) {
+    const modifiedViewIndexes = new Set(jsonViewIndexes);
+    if (jsonViewIndexes.has(index)) {
+      modifiedViewIndexes.delete(index);
+    } else {
+      modifiedViewIndexes.add(index);
+    }
+    setJsonViewIndexes(modifiedViewIndexes);
+  }
+
+  function handleDelete(index) {
+    setPastFiles([...pastFiles.slice(0, index), ...pastFiles.slice(index + 1)]);
+  }
+
+  return (
+    <>
+      <h1 className="font-bold text-4xl">Past Files</h1>
+      <p className="text-sm">These files are included within the longitudinal data</p>
+      <table className="w-full table-fixed text-left rounded shadow-md bg-gray-100">
+        <thead className="text-gray-500 font-thin rounded">
+          <tr>
+            <th scope="col" className="px-6 font-normal py-3">
+              Name
+            </th>
+            <th scope="col" className="font-normal py-3">
+              File size
+            </th>
+            <th scope="col" className="font-normal py-3">
+              Date added
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {pastFiles && pastFiles.length > 0 ? (
+            pastFiles.map((file, index) => (
+              <>
+                <tr key={file.name} className="bg-white border-b text-sm">
+                  <td className="px-6 text-base">
+                    <Icon icon="bi:file-earmark-code" className="text-4xl mr-2 my-2 inline-block text-gray-500" />
+                    {file.name}
+                  </td>
+                  <td className="text-gray-500">{file.size}</td>
+                  <td className="flex flex-row justify-between items-center my-4 h-full">
+                    <div className="text-gray-500">{file.dateAdded}</div>
+                    <div className="text-xs">
+                      <button
+                        onClick={() => toggleView(index)}
+                        type="button"
+                        className="font-bold text-blue-500 hover:text-blue-600"
+                      >
+                        {jsonViewIndexes.has(index) ? 'Hide' : 'View'}
+                      </button>
+                      <button
+                        onClick={() => handleDelete(index)}
+                        className="px-6 font-bold text-red-500 hover:text-red-600"
+                        type="button"
+                      >
+                        Delete
+                      </button>
+                    </div>
+                  </td>
+                </tr>
+                {jsonViewIndexes.has(index) ? (
+                  <tr>
+                    <td className="px-6" colSpan="3">
+                      <ReactJson src={file.body} collapsed={4} />
+                    </td>
+                  </tr>
+                ) : undefined}
+              </>
+            ))
+          ) : (
+            <tr>
+              <td colSpan="3" className="text-center bg-white italic">
```

Comment:
```suggestion
              <td colSpan="3" className="text-center bg-white italic my-2">
```

Let's add a bit of horizontal padding on this.